Xqsf 2 create works page

SatelliteSilver opened the pull request:
XQSF-2-create-works-page -> develop

SatelliteSilver committed:
XQSF-2-create-files

SatelliteSilver committed:
XQSF-2-create-works-banner

banner

SatelliteSilver committed:
XQSF-2-create-works-content

add a content to works page

SatelliteSilver commented:
.main_block 에 아래 옵션 필요할 듯
min-height: 100vh;
height: 100%;

yoonlight commented on css/works.css:
```diff
@@ -0,0 +1,53 @@
+    z-index: 1;
+}
+
+section {
```
class 명 추가 및 section -> .class 이름 형태로 변경

yoonlight commented on works.html:
```diff
@@ -0,0 +1,49 @@
+                    </div>
+                </div>
+            </div>
+            <section>
```
class 명 추가

yoonlight commented on css/works.css:
```diff
@@ -0,0 +1,53 @@
+.main__block {
+    display: none;
```
- class name 오타
- display: none 왜?
- min-height: 100vh 추가
- height: 100% 추가

SatelliteSilver replied:
display: none 엥 이게 왜 들어갔지,, 언제부터 들어간거지,, 뭔가 테스트하다가 안뺐나봄
삭제하였습니다

SatelliteSilver committed:
XQSF-2-add-section-class

SatelliteSilver committed:
XQSF-2-modify-mainblock-class

1. main__block -> main_block
2. delete the option display:none;
3. add options about height
